hydroflow_plus_kvs_server/src/main.rs

```diff
@@ -0,0 +1,26 @@
+use hydroflow_plus::hydroflow::bytes::Bytes;
```

Comment:
I can see there's three related crates here:
hydroflow_plus_kvs: defines the logic for the kvs.
hydroflow_plus_kvs_macro: staging helper macro.
hydroflow_plus_kvs_server: includes the hydroflow_plus_kvs outputs and actually calls them.

Do you intend on this pattern being common or is it just to demonstrate some scoping thing or that you can split them apart? I think it would be easier to understand if the whole hydroflow_plus_kvs_server was a #[test] in hydroflow_plus_kvs